packages/desktop-client/e2e/page-models/rules-page.ts

```diff
@@ -122,7 +122,15 @@ export class RulesPage {
 
       if (op && !fieldFirst) {
         await row.getByTestId('op-select').getByRole('button').first().click();
-        await this.page.getByRole('button', { name: op, exact: true }).click();
+        await this.page
+          .getByRole('button', { name: op, exact: true })
+          .first()
+          .waitFor({ state: 'visible' });
+
+        await this.page
+          .getByRole('button', { name: op, exact: true })
+          .first()
+          .click({ force: true });
```

Comment:
This is required as it looks like the new react-aria is rending it differently, potentially below another layer, preventing playwright from being "allowed" to click it.